src/containers/Session/SessionProvider.tsx

```diff
@@ -38,22 +39,22 @@ export const initialState: SessionState = {
 
 export interface SessionProps {
   userName?: string;
-  userAccess?: string;
+  userPermissions?: string[];
   authenticated: boolean;
   userNotRegistered: boolean;
-  login: (accessToken: string) => void;
+  login: (accessToken: Auth0DecodedHash) => void;
```

Comment:
Endre navn her slik at det passer med consten.
```suggestion
  login: (authResult: Auth0DecodedHash) => void;
```